src/main/java/com/squareup/javapoet/AnnotationSpec.java

```diff
@@ -122,6 +122,9 @@ public static Builder builder(Class<?> type) {
   public Builder toBuilder() {
     Builder builder = new Builder(type);
     builder.members.putAll(members);
+    for (Map.Entry<String, List<CodeBlock>> entry : builder.members.entrySet()) {
```

Comment:
Maybe add a `mutableMultimap()` method in `Util` to do just that? (reverse of `immutableMultimap()` as used in `build()`)